pkg/handlers/authorization/handler.go

```diff
@@ -83,7 +83,8 @@ func (auth *authorizationHandler) Process(req *http.Request) (*http.Request, err
 		projectNames := []string{}
 		projectUIDs := []string{}
 		for _, project := range projects {
-			projectNames = append(projectNames, project.Name)
+			quotedProjectName := `"`+project.Name+`"`
```

Comment:
instead of using concatenation like this can we instead use `fmt.Sprintf("%q", project.Name)` ?
we could even move that into the `append` directly like:
```
projectNames = append(projectNames, fmt.Sprintf("%q", project.Name))
```

Reply:
Yup. Fixed.